spec/features/jobs/user_creates_new_job_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe "User creates a new job" do
+  scenario "a user can create a new job" do
```

Comment:
same here - if a user doesn't enter in all the required attributes, we should have a test for that to ensure our user sees what we expect 👍